From e5b78fc3b610e20b3e26e05b3647a405c834c904 Mon Sep 17 00:00:00 2001
From: Bo Yang <bo.yang@amlogic.com>
Date: Sat, 21 Mar 2015 15:36:49 +0800
Subject: [PATCH 5487/5965] g9bb: t826: rename dtd for t826

differ from g9bb t828.

mesong9bb_pxp.dtd -> mesong9bb_t826_pxp.dtd
mesong9bb_skt.dtd -> mesong9bb_t826_skt.dtd

mk_g9bb.sh     -> mk_g9bb_n302.sh
mk_g9bb_skt.sh -> mk_g9bb_t826_pxp.sh
mk_g9bb_pxp.sh -> mk_g9bb_t826_skt.sh

Change-Id: If9cf1ebeebf9899063812f02f6702592f3a0821e
---
 arch/arm/boot/dts/amlogic/mesong9bb_n302.dtd  |  2 +-
 ...song9bb_pxp.dtd => mesong9bb_t826_pxp.dtd} |  0
 ...song9bb_skt.dtd => mesong9bb_t826_skt.dtd} | 30 ++++++++++++-------
 .../amlogic/{mk_g9bb.sh => mk_g9bb_n302.sh}   |  0
 .../{mk_g9bb_skt.sh => mk_g9bb_t826_pxp.sh}   |  6 ++--
 .../{mk_g9bb_pxp.sh => mk_g9bb_t826_skt.sh}   |  6 ++--
 6 files changed, 27 insertions(+), 17 deletions(-)
 rename arch/arm/boot/dts/amlogic/{mesong9bb_pxp.dtd => mesong9bb_t826_pxp.dtd} (100%)
 rename arch/arm/boot/dts/amlogic/{mesong9bb_skt.dtd => mesong9bb_t826_skt.dtd} (99%)
 rename scripts/amlogic/{mk_g9bb.sh => mk_g9bb_n302.sh} (100%)
 rename scripts/amlogic/{mk_g9bb_skt.sh => mk_g9bb_t826_pxp.sh} (67%)
 rename scripts/amlogic/{mk_g9bb_pxp.sh => mk_g9bb_t826_skt.sh} (67%)

diff --git a/arch/arm/boot/dts/amlogic/mesong9bb_n302.dtd b/arch/arm/boot/dts/amlogic/mesong9bb_n302.dtd
index 4aa5aece6918..f6624b40bd28 100644
--- a/arch/arm/boot/dts/amlogic/mesong9bb_n302.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9bb_n302.dtd
@@ -1,7 +1,7 @@
 /dts-v1/;
 /{
 	compatible = "AMLOGIC,T826_G9BB";
-	model = "AMLOGIC G9BB T826 For PXP";
+	model = "AMLOGIC G9BB T826";
 	interrupt-parent = <&gic>;
 	#address-cells = <1>;
 	#size-cells = <1>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9bb_pxp.dtd b/arch/arm/boot/dts/amlogic/mesong9bb_t826_pxp.dtd
similarity index 100%
rename from arch/arm/boot/dts/amlogic/mesong9bb_pxp.dtd
rename to arch/arm/boot/dts/amlogic/mesong9bb_t826_pxp.dtd
diff --git a/arch/arm/boot/dts/amlogic/mesong9bb_skt.dtd b/arch/arm/boot/dts/amlogic/mesong9bb_t826_skt.dtd
similarity index 99%
rename from arch/arm/boot/dts/amlogic/mesong9bb_skt.dtd
rename to arch/arm/boot/dts/amlogic/mesong9bb_t826_skt.dtd
index 39b9647fd49f..f123fa2991a0 100644
--- a/arch/arm/boot/dts/amlogic/mesong9bb_skt.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9bb_t826_skt.dtd
@@ -1,7 +1,7 @@
 /dts-v1/;
 /{
 	compatible = "AMLOGIC,T826_G9BB";
-	model = "AMLOGIC G9BB T826 For PXP";
+	model = "AMLOGIC G9BB T826 For SKT";
 	interrupt-parent = <&gic>;
 	#address-cells = <1>;
 	#size-cells = <1>;
@@ -255,7 +255,7 @@
 	uart_1 {
 		compatible = "amlogic,aml_uart";
 		port_name = "uart_b";
-		status = "okay";
+		status = "disabled";
 		dev_name = "uart_1";
 		pinctrl-names = "default";
 		pinctrl-0 = <&b_uart_pins>;
@@ -301,7 +301,7 @@
 //$$ L2 PROP_U32 2 = "key_tolerance"
 	adc_keypad {
 		compatible = "amlogic,adc_keypad";
-		status = "okay";
+		status = "disabled";
 		key_name = "enter", "left","right";
 		key_num = <3>;
 		key_code = <28 105 106>;
@@ -346,6 +346,19 @@
 
 
 
+/// ****************************************************************************
+//$$ DEVICE="wifi_power"
+//$$ L2 PROP_STR = "status"
+//$$ L2 PROP_STR = "power_gpio"
+	wifi_power {
+		compatible = "amlogic,wifi_power";
+		dev_name = "wifi_power";
+		status = "disable";
+		power_gpio = "GPIOY_5";
+	};
+
+
+
 
 /// ****************************************************************************
 ///	-	HDMI
@@ -357,7 +370,7 @@
 	amhdmitx {
 		compatible = "amlogic,amhdmitx";
 		dev_name = "amhdmitx";
-		status = "ok";
+		status = "disable";
 		vend-data = <&vend_data>;
 		pwr-ctrl = <&pwr_ctrl>;
 
@@ -418,7 +431,7 @@
 
 
 
-/// ***************************************************************************************
+/// ****************************************************************************
 ///	-	Securitykey
 //$$ MODULE="Securitykey"
 //$$ DEVICE="securitykey"
@@ -935,7 +948,7 @@
 	};
 
 
-/// ***************************************************************************************
+/// ****************************************************************************
 ///	-	USB Controller
 //$$ MODULE="USB Controller"
 	usb_con {
@@ -1122,7 +1135,7 @@
 			"LINEIN","Linein right 1";
 		pinctrl-names = "aml_snd_g9tv";
 		pinctrl-0 = <&audio_pins>;
-		status = "okay";
+		status = "disable";
 	};
 
 
@@ -1572,7 +1585,4 @@
 		pinctrl-0 = <&smc_pins>;
 	};*/
 
-
-
-
 }; /* end of root */
diff --git a/scripts/amlogic/mk_g9bb.sh b/scripts/amlogic/mk_g9bb_n302.sh
similarity index 100%
rename from scripts/amlogic/mk_g9bb.sh
rename to scripts/amlogic/mk_g9bb_n302.sh
diff --git a/scripts/amlogic/mk_g9bb_skt.sh b/scripts/amlogic/mk_g9bb_t826_pxp.sh
similarity index 67%
rename from scripts/amlogic/mk_g9bb_skt.sh
rename to scripts/amlogic/mk_g9bb_t826_pxp.sh
index d9e6f55615df..5b8f1b09287f 100755
--- a/scripts/amlogic/mk_g9bb_skt.sh
+++ b/scripts/amlogic/mk_g9bb_t826_pxp.sh
@@ -4,13 +4,13 @@
 make uImage -j8
 #make modules
 
-make mesong9bb_skt.dtd
-make mesong9bb_skt.dtb
+make mesong9bb_t826_pxp.dtd
+make mesong9bb_t826_pxp.dtb
 
 ROOTFS="rootfs.cpio"
 #ROOTFS="arch/arm/mach-mesong9bb/rootfs.cpio"
 
-./mkbootimg --kernel ./arch/arm/boot/uImage --ramdisk ./${ROOTFS} --second ./arch/arm/boot/dts/amlogic/mesong9bb_skt.dtb --output ./boot.img
+./mkbootimg --kernel ./arch/arm/boot/uImage --ramdisk ./${ROOTFS} --second ./arch/arm/boot/dts/amlogic/mesong9bb_t826_pxp.dtb --output ./boot.img
 ls -l ./boot.img
 echo "boot.img done"
 
diff --git a/scripts/amlogic/mk_g9bb_pxp.sh b/scripts/amlogic/mk_g9bb_t826_skt.sh
similarity index 67%
rename from scripts/amlogic/mk_g9bb_pxp.sh
rename to scripts/amlogic/mk_g9bb_t826_skt.sh
index 3ea93919df5b..6a3cddaa7002 100755
--- a/scripts/amlogic/mk_g9bb_pxp.sh
+++ b/scripts/amlogic/mk_g9bb_t826_skt.sh
@@ -4,13 +4,13 @@
 make uImage -j8
 #make modules
 
-make mesong9bb_pxp.dtd
-make mesong9bb_pxp.dtb
+make mesong9bb_t826_skt.dtd
+make mesong9bb_t826_skt.dtb
 
 ROOTFS="rootfs.cpio"
 #ROOTFS="arch/arm/mach-mesong9bb/rootfs.cpio"
 
-./mkbootimg --kernel ./arch/arm/boot/uImage --ramdisk ./${ROOTFS} --second ./arch/arm/boot/dts/amlogic/mesong9bb_pxp.dtb --output ./boot.img
+./mkbootimg --kernel ./arch/arm/boot/uImage --ramdisk ./${ROOTFS} --second ./arch/arm/boot/dts/amlogic/mesong9bb_t826_skt.dtb --output ./boot.img
 ls -l ./boot.img
 echo "boot.img done"
 
-- 
2.19.0

